.evergreen/aws-lambda-test/mongodb/src/main.rs

```diff
@@ -34,10 +34,12 @@ impl Stats {
 
     fn handle_sdam(&mut self, event: &SdamEvent) {
         match event {
-            SdamEvent::ServerHeartbeatStarted(_) => {
+            SdamEvent::ServerHeartbeatStarted(ev) => {
+                assert!(!ev.awaited);
```

Comment:
Rather than setting `patchable: true` so this is included in the evergreen runs and then (forget to) change it back before submitting, I just did an [independent run](https://spruce.mongodb.com/version/65ce39a6d6d80a58a79688fb/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC) for that.